components/TorchboxLogo.tsx

```diff
@@ -0,0 +1,40 @@
+import { Theme } from 'types/Base';
+
+type TorchboxLogoProps = {
+    theme: Theme;
+    className?: string;
+};
+
+export const TorchboxLogo = ({ theme, className }: TorchboxLogoProps) => (
+    <svg
+        className={className}
+        xmlns="http://www.w3.org/2000/svg"
+        width="137"
```

Comment:
**Though:** I tend to add [viewbox](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/viewBox) attributes manually  when they're missing, they can make it easier to size SVGs if needed in the future.